### PR TITLE
release step with file addition

### PR DESCRIPTION
**Pull request by cliffcaseyyet**

### CODE REVIEW

**Comment by kelvinqian00** [comment]
I suppose we can add two things in the documentation:
- Mention the bundle and what it contains in the README
- Add to the CHANGELOG.md file "Subsequent version release notes will be posted on the GitHub Releases page" as we deprecate said changelog

**Comment by cliffcaseyyet** [comment]
> I suppose we can add two things in the documentation:
> 
> * Mention the bundle and what it contains in the README
> * Add to the CHANGELOG.md file "Subsequent version release notes will be posted on the GitHub Releases page" as we deprecate said changelog

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -82,3 +82,25 @@ jobs:
...
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Download Bundle Artifact
+      uses: actions/download-artifact@v3
+      with:
+        name: datasim-bundle-artifact-${{ github.sha }}
+
+    - name: Craft Draft Release
+      uses: softprops/action-gh-release@v1
```
Comment (kelvinqian00) [inline .github/workflows/build.yml:100]: I just found out there's a v2 of the action-gh-release action ([see changelog](https://github.com/softprops/action-gh-release/blob/master/CHANGELOG.md))

Comment (kelvinqian00) [reply .github/workflows/build.yml:100]: On those same lines there's v4 of actions/checkout and actions/download-artifact.

Comment (kelvinqian00) [reply .github/workflows/build.yml:100]: Once we update those versions we should cut a v.x.x.x-pre version to ensure the artifact release works.

**Review by kelvinqian00 — APPROVED** [review]

**Review by milt — APPROVED** [review]

**Pull request merged**